ci/update-gh-pages.sh

```diff
@@ -85,8 +86,12 @@ jsduck \
     --output="$DOCS_W_EXT_DIR/" \
     "$DOWN_DIR/ext-$SENCHA_EXTJS_VERSION/packages/core/src" \
     "$DOWN_DIR/ext-$SENCHA_EXTJS_VERSION/classic/classic/src" \
+    --eg-iframe=docresources/eg-iframe.html \
```

Comment:
Can you please move `--eg-iframe`-line up after the `--output`-line?

This way the named and the positional arguments would be grouped together.